examples/grep-multiple-files/data/1.log

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
I think maybe let's drop this second example for now—it doesn't really tell us much more than we already know from the `ls` and `grep` examples.

Reply:
I've dropped it